app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -8,7 +8,7 @@ class AdministratorsController < Admin::BaseController
       before_action :load_admin, only: %i[edit update delete destroy]
 
       def index
-        @administrators = policy_scope(User).admins
+        @administrators = policy_scope(User).admins.order(:full_name)
```

Comment:
Is this case insensitive? Ideally it would be

Reply:
Yes, it's a db search and the default is case insensitive. I find it helpful to specify asc or desc regardless of the default

Reply:
Nope. How much do we care? It would become `order("LOWER(full_name)")` I think

Reply:
I don't think it matters _that_ much as hopefully people will be casing names correctly, but the lead providers page for example sorts "z" before "A" which is kinda weird

Reply:
Sorting on postgres is a bit of a minefield: https://dba.stackexchange.com/questions/106964/why-is-my-postgresql-order-by-case-insensitive
I'll leave it like this for now, and if people are putting in silly data we can always change it 🙂

Reply:
Or we could capitalize the name